packages/aws-cdk/test/integ/cli/test-cdk-ls.sh

```diff
@@ -15,6 +15,8 @@ ${STACK_NAME_PREFIX}-iam-test
 ${STACK_NAME_PREFIX}-lambda
 ${STACK_NAME_PREFIX}-missing-ssm-parameter
 ${STACK_NAME_PREFIX}-order-providing
+${STACK_NAME_PREFIX}-outputs-test-1
```

Comment:
Also make sure to add these new stacks to the cleanup function, I think....

Reply:
done